src/io/windows.zig

```diff
@@ -145,12 +145,10 @@ pub const IO = struct {
     fn flush_timeouts(self: *IO) ?u64 {
         var min_expires: ?u64 = null;
         var current_time: ?u64 = null;
-        var timeouts: ?*Completion = self.timeouts.peek();
 
         // iterate through the timeouts, returning min_expires at the end
```

Comment:
Unrelated, just spotted this (and many other non-tigerstyle comments) in this file!
Feel free to ignore this if you don't want to mix things.

```suggestion
        // Iterate through the timeouts, returning min_expires at the end.
```

Reply:
Will re-do comments in a separate PR!